code/clue.js

```diff
@@ -1,59 +1,238 @@
+  image: 'assets/mustard.png',
+  occupation: 'Colonel',
+  weapon: 'candlestick',
+  favouriteWeapon: 'knife'
+}
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: '#7ed4e6',
+  description: 'A grande dame, an elderly, attractive woman who nearly always maintains her dignity',
+  age: 63,
+  image: 'assets/peacock.png',
+  occupation: 'Socialite',
+  weapon: 'pistol',
+  favouriteWeapon: 'candlestick'
+}
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: '#75709e',
+  description: 'A quick-witted, professor with a bow tie and glasses',
+  age: 46,
+  image: 'assets/plum.png',
+  occupation: 'Professor',
+  weapon: 'poison',
+  favouriteWeapon: 'dumbbell'
+}
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: '#be2241',
+  description: 'A stock femme fatale, as young, cunning and attractive',
+  age: 28,
+  image: 'assets/scarlet.png',
+  occupation: 'Actor',
+  weapon: 'axe',
+  favouriteWeapon: 'poison'
+}
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: '#fef7fb',
+  description: 'A frazzled domestic sever, appears as a guest',
+  age: 37,
+  image: 'assets/white.png',
+  occupation: 'Maid',
+  weapon: 'rope',
+  favouriteWeapon: 'axe'
+}
+// OBJECTS FOR ALL THE WEAPONS
 
 const rope = {
-  name: 'rope',
-  weight: 10
+  name: 'Rope',
+  id: 'rope',
+  fact: 'Not for the clumsy ones!',
+  weight: 50,
+}
+const knife = {
+  name: 'Knife',
+  id: 'knife',
+  fact: 'Handle with care',
+  weight: 7,
+}
+const candlestick = {
+  name: 'Candlestick',
+  id: 'candlestick',
+  fact: 'Deadlier without a candle',
+  weight: 16,
+}
+const dumbbell = {
+  name: 'Dumbbell',
+  id: 'dumbbell',
+  fact: 'For the gym nut',
+  weight: 22,
+}
+const poison = {
+  name: 'Poison',
+  id: 'poison',
+  fact: 'Deadliest of all',
+  weight: .8,
+}
+const axe = {
+  name: 'Axe',
+  id: 'axe',
+  fact: 'Miss at first swing',
+  weight: 36,
+}
+const bat = {
+  name: 'Bat',
+  id: 'bat',
+  fact: 'A fool-proof weapon',
+  weight: 43,
+}
+const trophy = {
+  name: 'Trophy',
+  id: 'tropy',
+  fact: 'To celebrate the wins',
+  weight: 86,
+}
+const pistol = {
+  name: 'Pistol',
+  id: 'pistol',
+  fact: 'Loaded or not',
+  weight: 17,
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
-
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+// SUSPECTS, WEAPONS AND ROOMS IN ARRAYS:
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  colonelMustard,
+  mrsPeacock,
+  professorPlum,
+  missScarlet,
+  mrsWhite,
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+]
 
-const rooms = []
+const rooms = ['Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Billiard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'Guest House',
+  'Patio',]
```

Comment:
I suggest you remove the last comma after ‘Patio’ it may look like there could have been a deletions or something. This applies for all objects, for example also in line 173.